views/js/signup.js

```diff
@@ -1,12 +1,90 @@
 $(document).ready(function () {
   //init side bar
   $('.sidenav').sidenav();
+  M.AutoInit();
 
   var usernameBool = false;
   var emailBool = false;
   var passwordcharBool = false;
   var passwordconfirmBool = false;
 
+  // Live updates for the text fields
+  $('#username').on("input", function() {
+    if ($('#username').val().length >= 3) {
+      $('#username').removeClass("invalid");
+      $('#username').addClass("valid");
+    } else {
+      $('#username').removeClass("valid");
+      $('#username').addClass("invalid");
+    }
+  })
+  $('#username').blur(function() {
```

Comment:
*just a recommendation: If you want to keep the underline stay red, you can just do ` $('#LabelName').css('color', '#00bfa5');`

Reply:
I didn't change this due to keeping the class name determining the colour